mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraMessageMapper.java

```diff
@@ -328,6 +332,19 @@ private PropertyBuilder getPropertyBuilder(Row row) {
         return property;
     }
 
+    private List<AttachmentId> getAttachmentsIds(Row row, FetchType fetchType) {
+        switch (fetchType) {
+        case Full:
+        case Body:
+            return row.getList(ATTACHMENTS_IDS, String.class)
+                    .stream()
+                    .map(AttachmentId::from)
+                    .collect(org.apache.james.util.streams.Collectors.toImmutableList());
+        default:
+            return ImmutableList.of();
```

Comment:
I don't think we need to. It is the same principle in buildContent() for example.